include/zmq.h

```diff
@@ -368,7 +368,6 @@ ZMQ_EXPORT const char *zmq_msg_gets (const zmq_msg_t *msg, const char *property)
 #define ZMQ_VMCI_BUFFER_MAX_SIZE 87
 #define ZMQ_VMCI_CONNECT_TIMEOUT 88
 #define ZMQ_USE_FD 89
-#define ZMQ_BINDTODEVICE 90
```

Comment:
Perhaps add a comment that numbering continues elsewhere? Wouldn't an `enum` be better to avoid conflicts (e.g. two values later with same numbers)?

But even if so, this is a subject for subsequent PR :)